algorithms/src/std_algorithms/impl/Kokkos_RandomAccessIterator.hpp

```diff
@@ -184,6 +184,13 @@ class RandomAccessIterator< ::Kokkos::View<DataType, Args...> > {
   KOKKOS_FUNCTION
   reference operator*() const { return m_view(m_current_index); }
 
+  KOKKOS_FUNCTION
+  auto view() const {
```

Comment:
This is a bad name.  One could expect it returns `m_view`.

Reply:
Do you have some recommendation? 
Maybe better to suppress this method and do a check with`operator*()` alone?

Reply:
You could keep it returning `m_view` and only use it here for obtaining the stride.

Reply:
Sure. I have updated accordingly.